src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -207,6 +207,7 @@ internal static int GetWarningLevel(ErrorCode code)
             switch (code)
             {
                 case ErrorCode.WRN_AddressOfInAsync:
+                case ErrorCode.WRN_BadArgRef: // PROTOTYPE: Document in the markdown file.
```

Comment:
>ErrorCode.WRN_BadArgRef [](http://example.com/codeflow?start=21&length=23)

Why not use level 1 for this warning?

Reply:
Because it's used for pre-existing code... But that would be an errorneous code anyway, so it's fine to be level 1. Thanks.